java/configuration/src/main/java/sleeper/configuration/properties/instance/CdkDefinedInstancePropertyImpl.java

```diff
@@ -28,11 +28,13 @@ class CdkDefinedInstancePropertyImpl implements CdkDefinedInstanceProperty {
     private final String propertyName;
     private final String description;
     private final PropertyGroup group;
+    private final InstanceProperty defaultProperty;
```

Comment:
This field doesn't seem to be used. A CDK-defined property can't have a default value anyway.

Reply:
Again part of inheritance from the Instance Property. Removing this instance and assigning the method with an exception to be thrown.

Reply:
Addressed